timesketch/frontend/dist/index.html

```diff
@@ -1 +1 @@
-<!DOCTYPE html><html lang=en><head><meta name=csrf-token content="{{ csrf_token() }}"><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon href=/dist/favicon.ico><title>Timesketch</title><link href=/dist/css/chunk-common.045a93d9.css rel=preload as=style><link href=/dist/js/chunk-common.276d8a50.js rel=preload as=script><link href=/dist/js/chunk-vendors.1f9d0459.js rel=preload as=script><link href=/dist/js/index.0c45ccdf.js rel=preload as=script><link href=/dist/css/chunk-common.045a93d9.css rel=stylesheet></head><body><div id=app></div><script src=/dist/js/chunk-vendors.1f9d0459.js></script><script src=/dist/js/chunk-common.276d8a50.js></script><script src=/dist/js/index.0c45ccdf.js></script></body></html>
\ No newline at end of file
+<!DOCTYPE html><html lang=en><head><meta name=csrf-token content="{{ csrf_token() }}"><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon href=/dist/favicon.ico><title>Timesketch</title><link href=/dist/css/chunk-common.045a93d9.css rel=preload as=style><link href=/dist/js/chunk-common.276d8a50.js rel=preload as=script><link href=/dist/js/chunk-vendors.1f9d0459.js rel=preload as=script><link href=/dist/js/index.0c45ccdf.js rel=preload as=script><link href=/dist/css/chunk-common.045a93d9.css rel=stylesheet></head><body><div id=app></div><script src=/dist/js/chunk-vendors.1f9d0459.js></script><script src=/dist/js/chunk-common.276d8a50.js></script><script src=/dist/js/index.0c45ccdf.js></script></body></html>
```

Comment:
This should not be here but I can't find a way to sensibly remove it - permission is denied whenever I try to change it.

Reply:
It's fine, I'll rebuild the UI later.